source/userguide/internationalization.rst

```diff
@@ -0,0 +1,87 @@
+MindMeld for non-English languages
+==================================
+
+MindMeld supports the following languages:
+1. English - Supports Canada, United States and United Kingdom locales
+2. Danish
+3. Dutch
+4. Arabic
+5. French
+6. German
+7. Hungarian
+8. Italian
+9. Norwegian
+10. Portuguese
+11. Romanian
+12. Russian
+13. Spanish
+14. Swedish
```

Comment:
I think we should make the messaging here more nuanced. This makes it sound like only these languages will work, but these are just the languages where stemming works well, which is not important for many languages. e.g. the Vietnamese app seems to work pretty well and Vietnamese wouldn't benefit from stemming. I see it could be more complicated to the user but it would be more accurate to break it down by resource, something like:
* The current tokenizer works well for languages like English where words are delimited by spaces, but may be confused by punctuation marks not found in English. If the language you are dealing with does not use spaces between words or has many non-English-like punctuation marks, you should preprocess your data to remove punctuation and add spaces between words.
 * The stemmer supports the following languages [listed above]
If you are working with a language that is not supported by the stemmer, we recommend disabling the stemmer, increasing the use of character ngram features, and expect you will need more training data.
* The default system entity recognizer supports the following languages [listed]
If you are working with a language that is not supported by the default system entity recognizer, we recommend using custom entities and resolving the entities with custom logic in the dialogue manager.

Reply:
Addressed your comment

Reply:
I feel that apart from this PR we should do an evaluation of the effectiveness of these stemmers to provide the user with a reference of when/why they should be using stemmers.